app/controllers/callbacks_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CallbacksController < ApplicationController
```

Comment:
Update this controller in a way that we can add more providers in future like bitbucket with less changes